src/main/java/svc/profiles/LocalProfile.java

```diff
@@ -15,6 +15,9 @@
 	public DataSource dataSource() {
```

Comment:
You don't need this here. See the `application.yml` in uswide project. specify `spring.datasource.platform` to `local` and then you can use `data-local.sql`. Also, you shouldn't need `schema-tables.sql` since that should be auto-generated by hibernate.

Reply:
Correct.  I was testing with LocalWithHSQLAccessProfile.java mainly and had removed the datasource there but forgot to do so in the plain local profile.